packs/pagerduty/actions/ack_incident.yaml

```diff
@@ -0,0 +1,10 @@
+name: ack_incident
+runner_type: run-python
+description: ACK an incident on PagerDuty
+enabled: true
+entry_point: ack_incident.py
+parameters:
+  event_key:
+    type: string
```

Comment:
You can use type `list`. This way the command line client will do the splitting for you.

Please also change the key name to ``event_keys`` so it's explicit and implies it can contain more than one key.